shell/platform/common/accessibility_bridge.cc

```diff
@@ -111,6 +111,23 @@ AccessibilityBridge::GetPendingEvents() {
   return result;
 }
 
+void AccessibilityBridge::UpdateDelegate(
+    std::unique_ptr<AccessibilityBridgeDelegate> delegate) {
+  delegate_ = std::move(delegate);
+  // FlutterPlatformNodeDelegate[s] are created by the
+  // AccessibilityBridgeDelegate and may contain states from previous
+  // AccessibilityBridgeDelegate. Need to recreate them.
```

Comment:
Very pedantic wording tweak to clarify a little:
```
// Recreate FlutterPlatformNodeDelegates since they may contain stale state
// from the previous AccessibilityBridgeDelegate.
```